albumentations/augmentations/crops/transforms.py

```diff
@@ -392,49 +392,80 @@ class RandomCropNearBBox(DualTransform):
     """Crop bbox from image with random shift by x,y coordinates
 
     Args:
-        max_part_shift (float): float value in (0.0, 1.0) range. Default 0.3
+        max_part_shift (float, (float, float)): Max shift in `height` and `width` dimensions relative
+            to `cropping_bbox` dimension.
+            If max_part_shift is a single float, the range will be (max_part_shift, max_part_shift).
+            Default (0.3, 0.3).
+        cropping_box_key (str): Additional target key for croppting box. Default `cropping_bbox`
         p (float): probability of applying the transform. Default: 1.
 
     Targets:
         image, mask, bboxes, keypoints
 
     Image types:
         uint8, float32
+
+    Examples:
+        >>> aug = Compose(RandomCropNearBBox(max_part_shift=(0.1, 0.5), cropping_box_key='test_box'),
+        >>>               bbox_params=BboxParams("pascal_voc"))
+        >>> result = aug(image=image, bboxes=bboxes, test_box=[0, 5, 10, 20])
+
     """
 
-    def __init__(self, max_part_shift=0.3, always_apply=False, p=1.0):
+    def __init__(
+        self,
+        max_part_shift: Union[float, Tuple[float, float]] = (0.3, 0.3),
+        cropping_box_key: str = "cropping_bbox",
+        always_apply: bool = False,
+        p: float = 1.0,
+    ):
         super(RandomCropNearBBox, self).__init__(always_apply, p)
-        self.max_part_shift = max_part_shift
+        self.max_part_shift = to_tuple(max_part_shift, low=0)
```

Comment:
The docstring says that `If max_part_shift is a single float, the range will be (max_part_shift, max_part_shift).` but I assume this line will produce the range `(0, max_part_shift)` for a single float.

Reply:
Fixed